Source/ACE/Entity/Player.cs

```diff
@@ -1633,20 +1692,157 @@ public void HandleActionPutItemInContainer(ObjectGuid itemGuid, ObjectGuid conta
                     var motion = new UniversalMotion(MotionStance.Standing);
 
                     CurrentLandblock.EnqueueBroadcast(Location, Landblock.MaxObjectRange,
-                            new GameMessageUpdateMotion(Guid,
-                                                        Sequences.GetCurrentSequence(SequenceType.ObjectInstance),
-                                                        Sequences, motion));
+                        new GameMessageUpdateMotion(Guid,
+                            Sequences.GetCurrentSequence(SequenceType.ObjectInstance),
+                            Sequences, motion));
                     // CurrentLandblock.EnqueueBroadcast(self shame);
                 }
             });
 
             // Set chain to run
-            putItemInContainerChain.EnqueueChain();
+            pickUpItemChain.EnqueueChain();
+        }
+
+        private void SetInventoryForWorld(ObjectGuid containerGuid, WorldObject inventoryItem)
+        {
+            inventoryItem.Location = PhysicsData.Position.InFrontOf(1.1f);
+            inventoryItem.PositionFlag = UpdatePositionFlag.Contact
+                                         | UpdatePositionFlag.Placement
+                                         | UpdatePositionFlag.ZeroQy
+                                         | UpdatePositionFlag.ZeroQx;
+
+            inventoryItem.PhysicsData.PhysicsDescriptionFlag = inventoryItem.PhysicsData.SetPhysicsDescriptionFlag(inventoryItem);
+            inventoryItem.ContainerId = null;
+            inventoryItem.Wielder = null;
+            inventoryItem.WeenieFlags = inventoryItem.SetWeenieHeaderFlag();
+        }
+
+        private static void SetInventoryForOffWorld(ObjectGuid containerGuid, WorldObject inventoryItem)
+        {
```

Comment:
This method also belongs in WorldObject.  And it's more like "PutInContainer".  Also, containerGuid is not used.  Remove it or use it.